docs/source/starter/core_guide.rst

```diff
@@ -944,15 +944,15 @@ default implementation
 
 .. testcode::
 
-    def backward(self, use_amp, loss, optimizer):
```

Comment:
This shouldn't be deprecated. It's different from the attribute in the lightning module.

Reply:
Right, I know it's different but thought we didn't need this here as well. Will put back